scripts-components/toggler/src/index.ts

```diff
@@ -1,34 +1,26 @@
 import { Component, EventListeners, DelegateEvent } from '@mangoweb/scripts-base'
 
-// data-target
-// data-class-name
-// data-toggle-others
-// data-prevent-default
-
-export default class Toggler extends Component<void> {
+export default class Toggler extends Component {
 	static componentName = 'Toggler'
 
 	getListeners = (): EventListeners => [['click', '.toggler', this.handleClick]]
```

Comment:
Já si pořád myslím, že by to `.toggler` i `.is-active` mělo být konfigurovatelné přes props. :neutral_face:

Reply:
@LukasChalupa, `.toggler` a `.is-active` můžeš nechat třeba jako default, aby ti dál fungovaly starší projekty. `this.getPropOrElse('togglerSelector', '.toggler')`

Reply:
Nějak pocitově jsem usoudil, že tady ta konfigurovatelnost tak důležitá nění :). Ona usage týhle komponenty měla být, že jí jednou nainicializuješ na body a pak jí jen používáš přidáním té třídy `.toggler` a data atributů. To `.is-active` je jen fallback, když nenastavíš `data-class-name`.

Ale nejspíš máš pravdu, ještě to předělám, aby to šlo konfigurovat.

Reply:
@JirkaVebr tak už :) hodil jsem to do props